src/System.Runtime.Extensions/src/System/Environment.Unix.cs

```diff
@@ -36,6 +37,7 @@ public static partial class Environment
             }
             return results;
         });
+        private static readonly bool IsMac = Interop.Sys.GetUnixName() == "OSX";
```

Comment:
> RuntimeInformation APIs better because a platform can be both Unix and Mac.

What API on RuntimeInformation tells you whether something is Unix (other than !Windows, which can be done with OSVersion as well)?